docs/how-to/modify-kernel-boot-parameters.rst

```diff
@@ -0,0 +1,140 @@
+How to modify kernel boot parameters
+=======================================
+
+The real-time performance-tuning features compiled into the kernel can be
+controlled by various means. The approach used here is a common one:
+passing command line parameters via a bootloader.
+
+The Linux kernel supports a wide range of parameters, but only a few of them are relevant to real-time tuning.
+:doc:`/reference/kernel-boot-parameters` is good a reference.
+
+Depending on the underlying boot mechanism, there are multiple ways to pass command line parameters to the kernel.
+
+For Raspberry Pi, jump to :ref:`raspberry-pi`.
```

Comment:
```suggestion
- For Raspberry Pi, jump to :ref:`raspberry-pi`.
- Ubuntu Core uses GRUB but provides a different interface for...
```

Suggesting to indent this for readability because you mentioned "multiple ways to pass command line parameters" and these are the 2 methods you are referring to.

Reply:
I agree with the poor readability, but I don't feel bullets make it better:

![image](https://github.com/canonical/real-time-ubuntu-docs/assets/11150423/94732468-a6fa-438d-b28c-94011934b046)

vs.

![image](https://github.com/canonical/real-time-ubuntu-docs/assets/11150423/ad2e2095-bb8e-49a6-a700-5d3574ac8ba3)

What do you think?

Reply:
I think the second option is better. The bullets (or rather the indentation) makes it a bit more obvious that these are the 3 different methods, depending on your system.

thanks!

Reply:
Ok, changed. Also added a link to the grub section.